payments-core/src/main/java/com/stripe/android/view/PaymentAuthWebViewClient.kt

```diff
@@ -184,7 +184,8 @@ internal class PaymentAuthWebViewClient(
 
         private val COMPLETION_URLS = setOf(
             "https://hooks.stripe.com/redirect/complete/",
-            "https://hooks.stripe.com/3d_secure/complete/"
+            "https://hooks.stripe.com/3d_secure/complete/",
+            "https://hooks.stripe.com/3d_secure_2/hosted/complete"
```

Comment:
This may be for a future PR, but can't we just match all urls that are `https://hooks.stripe.com`, then we don't need to maintain this list?

Reply:
There's also `https://hooks.stripe.com/three_d_secure/authenticate` which triggers a different flow. So unfortunately not :(

Reply:
This does feel like a value that should come from a server rather than be hard coded though. Hopefully we can add it later this year after we have that capability.